x/posts/keeper/handler.go

```diff
@@ -115,6 +126,16 @@ func handleMsgEditPost(ctx sdk.Context, keeper Keeper, msg types.MsgEditPost) (*
 	existing.Message = msg.Message
 
 	if msg.Attachments != nil {
+		// Check if any of the tags have blocked the post creator
+		for _, attachment := range msg.Attachments {
+			for _, tag := range attachment.Tags {
+				// check if the request's receiver has blocked the sender before
+				if keeper.IsUserBlocked(ctx, tag, existing.Creator) {
+					return nil, sdkerrors.Wrap(sdkerrors.ErrInvalidRequest,
+						fmt.Sprintf("The user with address %s has been blocked from %s", existing.Creator, tag))
+				}
+			}
+		}
```

Comment:
I would place this piece of code inside an external method since it's also used inside the `MsgCreatePost` handler